src/app/views/orders/order.controller.coffee

```diff
@@ -94,7 +94,11 @@
             toastr.success('mnoe_admin_panel.dashboard.subscriptions.modal.approve_subscriptions.toastr_success', {extraData: {subscription_name: vm.order.product.name}})
           ->
             toastr.error('mnoe_admin_panel.dashboard.subscriptions.modal.approve_subscriptions.toastr_error', {extraData: {subscription_name: vm.order.product.name}})
-        )
+        ).then(() ->
+            MnoeProvisioning.getSubscriptionEvents(vm.orderId, vm.orgId)
```

Comment:
Can we refactor this into a function like `refreshSubscriptionEvents` maybe? And then just pass that function to both `.then`s in this controller.

Reply:
Ya probably better